extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/conf/ConfigurationService.java

```diff
@@ -85,4 +85,34 @@ public PrivateKey getClearpassKey() throws GuacamoleException {
         return environment.getProperty(CASGuacamoleProperties.CAS_CLEARPASS_KEY);
     }
 
+    /**
+     * Returns the attribute used to determine group memberships
+     * in CAS, or null if not defined.
+     *
+     * @return
+     *     The attribute name
```

Comment:
This should probably be more along the lines of "The attribute name used to determine group memberships in CAS, or null if not defined."  I know it seems redundant based on what's already a couple of lines above it, but when people are looking at the JavaDoc documentation for a Guacamole method there are situations where it can be useful.